packages/components/src/List/ListItemLayout.tsx

```diff
@@ -86,7 +86,8 @@ export const listItemIconCSS = css<ListItemLayoutProps>`
 
 export const ListItemLayout = styled(ListItemLayoutInternal).attrs(
   ({ color, disabled }) => ({
-    color: disabled ? 'text1' : color || 'text1',
+    // Determines icon color
+    color: disabled ? 'text1' : color || 'text2',
   })
```

Comment:
I know the general structure here was already set, but I can see it's approaching a nested ternary in that it's probably easy to write but unnecessarily hard to read. I might just update the destructuring assignment to set the default value there:

```suggestion
 ({ color = 'text2', disabled }) => ({
    color: disabled ? 'text1' : color,
  })
```

CC @lukelooker since we were just talking about trying to remove all our nested ternaries yesterday. This one is kind of a grey area, but I think trying avoid multiple levels of nesting logic in general is probably a good rule of thumb

Reply:
Yeah this suggestion looks a lot cleaner than having both the ternary and the or operator there. Good catch.

Reply:
Resolved in e06184b